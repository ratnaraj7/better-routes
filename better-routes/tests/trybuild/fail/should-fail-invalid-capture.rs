use axum_extra::routing::RouterExt;
use better_routes::{method_helper, routes};
use serde::Deserialize;
#[derive(Deserialize)]
struct Foo;
#[method_helper]
impl Foo {}
routes! {
    name => FooPath,
    "/:id" => Foo
}
fn main() {}
