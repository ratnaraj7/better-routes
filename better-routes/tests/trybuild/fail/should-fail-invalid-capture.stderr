error[E0026]: struct `Foo` does not have a field named `id`
  --> tests/trybuild/fail/should-fail-invalid-capture.rs:10:5
   |
10 |     "/:id" => Foo
   |     ^^^^^^ struct `Foo` does not have this field
